src/elements/content-sidebar/versions/VersionsItemActions.scss

```diff
@@ -0,0 +1,37 @@
+@import '../../common/variables';
+
+.bcs-VersionsItemActions-item {
+    .fill-color {
+        fill: $fours;
+    }
+
+    .stroke-color {
+        stroke: $fours;
+    }
+}
+
+.bcs-VersionsItemActions-toggle {
+    align-items: center;
+    align-self: flex-start;
+    border-radius: 2px;
+    display: flex;
+    height: 24px;
+    justify-content: center;
+    outline: none;
+    pointer-events: auto;
+    width: 32px;
+
+    &:focus,
+    &:hover {
+        background-color: $off-white;
+        fill: $fours;
+    }
+
+    .fill-color {
+        fill: $nines;
+    }
+
+    .icon-clock-past {
+        margin-left: -2px;
```

Comment:
What is this margin adjustment used for?

Reply:
The clock icon has a slightly different size and viewbox than the other icons, so it needed a little extra love.